include/tvSettingsExtODM.h

```diff
@@ -59,33 +59,6 @@ extern "C"
 {
 #endif
 
-/**
- * @brief Reads PQ capability information sections PQ capability configuration file. To be deprecated soon.
```

Comment:
Is this expected to be removed part of this PR? If so, please include it in the issue description.

Reply:
It already removed as part of Phase-1 ODM removal.
I have updated this PR. This error wont be seen on latest commit.